src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -336,7 +338,7 @@ export default class TaskRoleContainerList extends React.Component {
     const {className, style, taskInfo} = this.props;
     const status = isNil(taskInfo) ? this.generateDummyTasks() : taskInfo.taskStatuses;
     return (
-      <div className={className} style={{backgroundColor: theme.palette.white, ...style}}>
+      <div className={className} style={{backgroundColor: 'theme.palette.white', ...style}}>
```

Comment:
`theme.palette.white` is a javascript string variable. It doesn't need quotation marks